Platforms/OvmfPkg/OvmfPkgX64.fdf

```diff
@@ -115,6 +115,7 @@ SET gUefiCpuPkgTokenSpaceGuid.PcdSevEsWorkAreaSize = gUefiOvmfPkgTokenSpaceGuid.
 ################################################################################
 
 [FV.SECFV]
+FvForceRebase      = FALSE
```

Comment:
@TaylorBeebe @spbrogan Is this right? If I do not add them here, the alignment flag from X64 tools def will make the GenFv puke with "PE image Section-Alignment and File-Alignment do not match" on the PEI and SEC modules.

Reply:
I ran into this a few months ago when trying to align PEI modules and started digging into it but was stopped by Sean. I think the reason he gave was that we would never allow the space bloat of aligning PEI/SEC modules. However, wouldn't SEC and PEI modules be built with the IA32 flags?

Reply:
This specific platform build is entirely based on x64 flags, as indicated in the file name. I agree that PEI/SEC should not expand size, but in that case, we need to override the build options for SEC and PEI modules. Do you have any suggestions on how to do that?

Reply:
Well, reverted that as we do not need to build them anymore.